content/applications/hr/payroll.rst

```diff
@@ -108,6 +108,17 @@ The contract template forms the basis of an offer, and can be modified for speci
 employees, when necessary. If a contract template is created or modified in the *Payroll*
 application, the changes are also reflected in the *Recruitment* application.
 
+.. important::
+   To access contract templates, the :guilabel:`Salary Configurator` module **must** be installed.
```

Comment:
```suggestion
   To access contract templates, the *Salary Configurator* module **must** be installed.
```
It's my understanding that we should italicize modules the same way we do with application names.